graphqlapi/src/main/java/io/stargate/graphql/schema/CustomScalar.java

```diff
@@ -78,14 +78,20 @@
       "BigInt",
       "The `BIGINT` scalar type represents a CQL bigint (64-bit signed integer) as a string.",
       e -> {
+        if (e instanceof String) {
+          return new Long((String) e);
+        }
         if (e instanceof Long) {
-          return BigInteger.valueOf((Long) e);
-        } else if (e instanceof BigInteger) {
           return e;
         }
-        return null;
+        if (e instanceof Integer) {
+          return ((Integer) e).longValue();
+        }
+        throw new NumberFormatException(
+            String.format(
+                "Expected string for bigint scalar, obtained %s", e.getClass().getName()));
       },
-      e -> e.toString()),
+      Object::toString),
```

Comment:
I think we can reuse `graphql.Scalars.GraphQLLong` instead of building our own custom type. It would work for CQL type `counter` as well.

`graphql.Scalars` also defines `GraphQLShort`, `GraphQLByte`, `GraphQLBigDecimal` and `GraphQLBigInteger`, that all map directly to their CQL counterparts.

Reply:
oh good tip, TIL those java representations of custom scalars existed, I've filed #272 that can be implemented w/ #261 .